src/ts/components/seekbar.ts

```diff
@@ -520,6 +579,8 @@ export class SeekBar extends Component<SeekBarConfig> {
     let seekBarContainer = new DOM('div', {
       'id': this.config.id,
       'class': this.getCssClasses(),
+      'role': 'slider',
+      'tabindex': this.config.tabindex.toString(),
```

Comment:
`toString()` on a `number`??

Reply:
`string is not assignable to number`

Reply:
It looks to me that you are calling `toString()` on a `number` (which is a primitive, not an object with functions). Is `this.config.tabindex` not a `number`?

Reply:
It is a number but `DOM` accepts only strings as values

Reply:
👍